pkg/utils/utils.go

```diff
@@ -114,3 +120,66 @@ func Int32Ptr(x int32) *int32 {
 func AddDryRunPrefix(msg string) string {
 	return fmt.Sprintf("[DRY RUN] %s", msg)
 }
+
+func UploadFile(url, filePath string, formFields map[string]any) error {
+	buffer := new(bytes.Buffer)
+	writer := multipart.NewWriter(buffer)
+
+	fileInfo, err := os.Stat(filePath)
+	if err != nil {
+		return err
+	}
+
+	if fileInfo.Size() > maxFileSize {
+		return fmt.Errorf("File size exceeds the limit of 1GB. Actual size: %d", fileInfo.Size())
```

Comment:
Can we format this size instead of printing the number of bytes?
```suggestion
		return fmt.Errorf("file size %d exceeds the 1GB limit", fileInfo.Size())
```